tests/cyr_narcom_to_cyr/test_cases/test_1.json

```diff
@@ -0,0 +1,19 @@
+{
+  "tests": [
+    {
+      "name": "test_1",
+      "input": "FrGpMnWmcTq3",
+      "expected_output": "frgpmnwmctq3"
+    },
```

Comment:
Regarding test structure. It seems like we'll need to support at least 3 different types of text: narkam, tarashk and lacinka. What do you think if test case contains potentially all three? For example:

```json
{
  "name": "some name",
  "cyr_nar": "снег",
  "cyr_taras": "сьнег",
  "lac": "śneh",
}
```

And then libraries can decide what they want to test. Or probably test multiple variations for each test.